cmd/sql/flow.go

```diff
@@ -503,27 +502,58 @@ func executeDeployCmd(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	dagsPath := filepath.Join(projectDir, ".deploy/"+env+"/dags")
-	if err := os.MkdirAll(dagsPath, os.ModePerm); err != nil {
-		return fmt.Errorf("error creating directories for %v: %w", dagsPath, err)
+	dagsPath := filepath.Join(config.WorkingPath, "dags/.astro-sql-deploy/"+env)
```

Comment:
Yes, I like that. I mean either way works for me. We can also add an optional flag maybe? so that the user can decide whether he wants to `--clear-existing` or not.

Reply:
That's a great suggestion, @dimberman , could you do this? Perhaps by default, we should clean and have a `--keep-dags` in case the user wants to keep them. We can review this based on user feedback.

Reply:
I've added a delete-dags flag